web/src/message_viewport.js

```diff
@@ -481,7 +480,7 @@ export function keep_pointer_in_view() {
 
 export function initialize() {
     $jwindow = $(window);
-    $message_pane = $(".app");
+    $message_pane = $("html");
```

Comment:
This can be a follow-up, but probably we're going to want to rename this variable to reflect the fact that it's the scrolling container, not the message pane, at this point.